### DISCUSSION TITLE
What's the best way to enable eufy2c (non-continuous) battery camera's in Frigate?

### DISCUSSION THREAD

**Discussion by atv2016** [blakeblackshear/frigate#5731] [General]
Just wondering, would be nice to have them, a black screen until a feed shows up is fine.
But ffmpeg might go crazy trying to find them if they don't show up..

**Comment by mrplow** [comment]
Agreed. It would be nice to have the option to disable a camera.

I have the same issue. I have an indoor camera that I only plug in when leaving for extended periods of time to watch the cat and dog.
I comment out the relevant config sections for that camera when it's not in use

**Reply by NickM-27** [reply]
In 0.12 there is a new `enabled:` tag in the config to make this easier.

**Reply by atv2016** [reply]
Im just disabling most of the logs for that specific eufy camera for now, apart from some go2rtc logs that it can’t reach it it works amazingly well. Just shows a black screen until it triggers.

**Comment by atv2016** [comment]
One thing that probably needs adjusting is that sometimes events for eufy battery cameras seem to keep being in the “in progress” state. I guess it switches off before it sees the person move out of the frame or something like that.

Any way to combat that configuration wise?
________________________________

**Reply by NickM-27** [reply]
Set a `max_frames` value for `detect -> stationary `
